app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -16,11 +16,11 @@
         = render "publishers/vacancies/vacancy_form_partials/hidden_state_input", f: f
 
         - if JobseekerApplicationsFeature.enabled?
-          = f.govuk_radio_buttons_fieldset :apply_through_teaching_vacancies do
-            = f.govuk_radio_button :apply_through_teaching_vacancies, :yes, link_errors: true do
+          = f.govuk_radio_buttons_fieldset :enable_job_applications do
+            = f.govuk_radio_button :enable_job_applications, true, link_errors: true do
               = f.govuk_text_area :personal_statement_guidance, label: { size: "s" }
 
-            = f.govuk_radio_button :apply_through_teaching_vacancies, :no do
+            = f.govuk_radio_button :enable_job_applications, "false" do
```

Comment:
why "false" is a string here?

Reply:
It wasn't working with literal `false` – I _think_ because it's falsey the form builder doesn't pick it up (not a problem for `true` 😕 

Another option would be "0" and "1", but I think I prefer it this way even if it's a bit ugly...